gpkit/interactive/ractor.py

```diff
@@ -10,23 +10,27 @@
 
 from widget import widget
 
+
 def showcadtoon(title, css=""):
     with open("%s.gpkit" % title, 'r') as f:
         css = "<style> #ractivecontainer { %s } </style>" % css
         display(HTML(f.read() + css))
 
-def ractorpy(gp, update_py, ranges, constraint_js="",
+
+def ractorpy(m, update_py, ranges, constraint_js="",
              showtables=["cost", "sensitivities"]):
-    def ractivefn(gp):
-        live = "<script>" + update_py(gp) + "\n" + constraint_js + "</script>"
+    def ractivefn(sol):
+        live = "<script>" + update_py(sol) + "\n" + constraint_js + "</script>"
         display(HTML(live))
         if showtables:
             print sol.table(showtables)
-    return widget(gp, ractivefn, ranges)
+    return m.interact(ractivefn, ranges)
+
 
 new_jswidget_id = itertools.count().next
 
-def ractorjs(title, gp, update_py, ranges, constraint_js=""):
+
+def ractorjs(title, m, update_py, ranges, constraint_js=""):
```

Comment:
the signature of this method is so similar to `ractorpy` -- could we make `title` a keyword arg so that they actually do have the same args?